.github/workflows/push.yml

```diff
@@ -4,9 +4,7 @@ name: Build and push Docker Image
 on:
   push:
     branches-ignore:
-      - main
-      - master
-      - dependabot/*
+      - 'dependabot/**'
```

Comment:
No ' unsed at it
```suggestion
      - dependabot/**
```

Reply:
No idea how that was added, probably some misconfigured linter or something like this. It's now removed.